fixtures/Program.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Linq;
```

Comment:
Is this used?

Reply:
Yes, it adds `Where` extension to `IList`.

Reply:
Ugh. Implicit side-effects are the worst.  :)